module/data.js

```diff
@@ -6,6 +6,7 @@ export const exercises = {
   obj: { feedback: true },
   nested: { feedback: true },
   mutability: { feedback: true },
+  'molecules-cells': { feedback: true },
```

Comment:
what made you pick this place in the module ?
It seems very very early for me at this point to ask this.

I'll place it where I feel it fit better, feel free to explain your decision.